03week/ticTacToe.js

```diff
@@ -25,69 +22,154 @@ function printBoard() {
 
 function horizontalWin() {
   // Your code here
+  if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn ||
+      board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn ||
+      board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn ||
+      board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn ||
+      board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn ||
+      board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn ||
+      board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn ||
+      board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn) {
+        console.log(playerTurn + '\'s Win!!!');
+          reset();
```

Comment:
you should break these up into their own functions. Horizontal, vertical and diagonal wins, like the tests require.